app/Http/Controllers/StoreController.php

```diff
@@ -115,7 +115,9 @@ public function getCheckout()
 
         $addresses = Auth::user()->storeAddresses()->with('country')->get();
 
-        return view('store.checkout', compact('order', 'addresses'));
+        $delayedShipping = Store\Order::where('orders.status', 'paid')->count() > env('DELAYED_SHIPPING_ORDER_THRESHOLD', 100);
```

Comment:
I vaguely remember having some kind of pain figuring out environment variables outside of config files so I usually prefer this flow

```
env -> config -> app
```